src/CMakeLists.txt

```diff
@@ -43,7 +43,7 @@ set(LIBRARY_SOURCES
   ${HEADERS}
   )
 
-set(PROGRAM "cmark")
+set(PROGRAM "cmark_exe")
```

Comment:
why the name change?

Reply:
the `cmark_exe` never leaves the project, unlike the library target `cmark` which is exported into `cmark-config.cmake`